content/docs/atom-guide/look-dev/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,716 @@
+   worldPosition.y += MaterialSrg::m_yOffset;
+   ```
+   Recall that we added the parameters into the material shader resource group, so that's how we can reference them with MaterialSrg.
+3. Repeat with the depth pass.
+4. Save your files and open the **Material Editor**.
+5. Select the material you made with the VegetationBending material type previously (`my_material`), and find **Vegetation Bending** in the **Inspector** on the right. Adjust the x and y offsets as you see fit!
+6. Save and return to the **Editor**.
+7. Observe how the offset matches your inputs from the **Material Editor**!
+
+{{< image-width src="/images/atom-guide/vegetation-bending-tutorial/parameteroffset.png" width="100%" alt="Parameter offset added." >}}
+
+Congrats! Now you have taken the first step to writing your own custom shaders.
+
+## Prepare to add vegetation bending
+Now that we've edited our vertex shader, let's take the next step and prepare to add vegetation bending by adding an appropriate model.
+
+### Add a tree
+In order to test the code that we are about to write, we need an appropriate mesh!
+
+1. Download `tree.fbx` from here.
+2. Place `tree.fbx` in `{your_project_path}\Objects`. Make the `Objects` folder as needed.
+3. Open the **Editor** to your project and level. 
+4. On the left in the **Entity Outliner**, **right-click** and select **Create entity**. **Right-click** the new entity in the **Entity Outliner**, and rename the entity to `Tree`.
+5. On the right, in the **Entity Inspector** click **Add Component** and select **Mesh**.
+6. In the mesh component, click the file icon next to **Model Asset** select `tree.fbx` in your project's `Objects` folder.
+7. Again in the mesh component, click **Add Material Component**, click the file icon next to *Default Material* and select your material that you made earlier (`my_material`).
+
+Now, we have a tree (at an offset)! This tree is important because it uses vertex colors that we will be using to determine how the tree should bend.
+
+Note on vertex colors: You can use a DCC tool to color the vertices to match the type of bending you want to do on each part of the tree: 
+- Red vertex color: smaller movement with high frequency of random sinusoidal noise.
+- Green vertex color: delays the start of the movement for variations with high frequency of random sinusoidal noise.
+- Blue vertex color: larger movement and bending with low frequency of sinusoidal noise. 
+In the case of our tree, the trunk's vertices are colored blue and leaves are colored red.
+
+### Add a shader option
+Our tree mesh that we just added has colored vertices; however, not every mesh that you want to use your material type on may have colored vertices. Therefore, we want to add a [*shader option*](../dev-guide/shaders/azsl/#shader-variant-options) that allows the vertex shader to handle both of these cases. We also want to add the reference to the colors so we can use them in the code.
```

Comment:
nit: I think 'apply your material type to' is clearer than 'use your material type on', but that might just be me.